Index: server.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server.py b/server.py
--- a/server.py	
+++ b/server.py	
@@ -17,7 +17,7 @@
         self.addreses = []
         self.output_addresses = output_addresses
         self.output_port = output_port
-        self.HOST = "26.36.124.241"  # Standard loopback interface address (localhost)
+        self.HOST = "26.181.96.20"  # Standard loopback interface address (localhost)
         self.PORT = server_port  # Port to listen on (non-privileged ports are > 1023)
         self.server = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         self.CHUNK = 4096
@@ -28,8 +28,8 @@
     async def read_request(self):
         self.server.bind((self.HOST, self.PORT))
         self.data, self.address = self.server.recvfrom(self.CHUNK)
-        print(self.address[0])
-        c = self.output_addresses.index[self.address[0]]
+        print(self.output_addresses.index())
+        c = self.output_addresses.index(self.address[0])
         del self.output_addresses[c]
         print(f"Connected to: {self.address}")
         cl = Client(address=self.address)
